service/src/main/java/com/ericsson/eiffel/remrem/generate/controller/RemremGenerateController.java

```diff
@@ -97,10 +97,12 @@ public void setRestTemplate(RestTemplate restTemplate) {
             @ApiParam(value = "message type", required = true) @RequestParam("msgType") final String msgType,
             @ApiParam(value = "ER lookup result multiple found, Generate will fail") @RequestParam(value = "failIfMultipleFound", required = false, defaultValue = "false") final Boolean failIfMultipleFound,
             @ApiParam(value = "ER lookup result none found, Generate will fail") @RequestParam(value = "failIfNoneFound", required = false, defaultValue = "false") final Boolean failIfNoneFound,
+            @ApiParam(value = RemremGenerateServiceConstants.SHALLOW) @RequestParam(value = "shallow", required = false, defaultValue = "true") final Boolean shallow,
+            @ApiParam(value = RemremGenerateServiceConstants.PAGESIZE) @RequestParam(value = "pageSize", required = false, defaultValue = "1") final int pageSize,
```

Comment:
I would like to see the default values of all these parameters taken from a config file instead of just hard coded in the code. Could you please make it possible to set these 4 parameters through application.properties?

Reply:
Checking on this whether its possible or not